CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## Next
 
+### Tasks
+* [#2225](https://github.com/dso-toolkit/dso-toolkit/issues/2225) DangerJS in CI
```

Comment:
Let op: De formatting in CHANGELOG verandert! Waar voorheen het linkbare issue id achteraan tussen haakjes staat is deze naar het begin zonder haakjes verplaatst.

Reply:
Zou voor onze klanten wel wat minder overzichtelijk kunnen zijn, want het #-nummer zal ze niet zo veel zeggen, en die komt nu als eerste. We gaan het zien, misschien moeten ze gewoon de blogpost lezen :)

Reply:
Mijn overwegingen:
- Dat de issue nummers dan uitlijnen terwijl ze nu aan het einde van de zin staan;
- Overal staan de issue nummers vooraan BEHALVE in de CHANGELOG;

Maar toen ik met een ander PR bezig was begon ik alweer te twijfelen.. 😩 Niet veranderen?

Reply:
Ik zou het gewoon doen. Iets met toestemming en vergiffenis 😉

Reply:
Ja.